### PR TITLE
adding teacher for academic course from subject

### PR DESCRIPTION
**Pull request by EdgarTsarukyan**

### CODE REVIEW

File: src/main/java/com/epam/edumanagementsystem/admin/model/dto/AcademicCourseDto.java [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
 
 public class AcademicCourseDto {
     private Long id;
     private String name;
     private Subject subject;
+    private Set<Teacher> teacher = new HashSet<>();
```
Comment (ArmineZak) [inline src/main/java/com/epam/edumanagementsystem/admin/model/dto/AcademicCourseDto.java]: Assigning a value to a field like this is considered a bad practice, you could assign a new HashSet to teacher variable in constructor.

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -6,8 +6,8 @@ spring:
   datasource:
     platform: postgres
     driverClassName: org.postgresql.Driver
-    url: jdbc:postgresql://edu-db:5432/edumanagement
-    username: root
+    url: jdbc:postgresql://localhost:5432/edumanagement
+    username: postgres
```
Comment (ArmineZak) [inline src/main/resources/application.yml]: Please,stop commiting and pushing changes in yml file to 3Gteam branch.Instead just untic it from the list of the commited files.

File: src/main/resources/templates/academicCourseSectionForTeachers.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+</head>
+<body>
+<div class="header" id="header">
+    <div th:insert="fragments/fragments :: container"></div>
+</div>
+<div class="main-wrapper" id="main-wrapper">
+    <div class="sidebar" th:insert="fragments/fragments :: menu"></div>
+    <div class="middle" id="middle">
+        <div class="url-path">
+            <a class="a-url" th:href="${'/'+firstUrl}" th:text="${'Academic Course'}"></a>
```
Comment (ArmineZak) [inline src/main/resources/templates/academicCourseSectionForTeachers.html:23]: When each part of the url is longer than specific number of symbols, it is not shown in one line any longer, but is returned in two lines.

File: src/main/resources/templates/academicCourseSection.html [hunk 4]
```diff
@@ -56,7 +56,7 @@ <h1>New Academic course</h1>
                             th:field="*{subject}">
                         <option  th:value="*{subject}"></option>
                         <option  th:each="subject : ${subjects}" th:value="${subject.id}"
-                                th:text="${subject.getName()}"></option>
+                                 th:text="${subject.getName()}"></option>
                     </select>
                     <div class="error" th:if="${#fields.hasErrors('subject')}"
                          th:errors="*{subject}"></div>
```
Comment (ArmineZak) [inline src/main/resources/templates/academicCourseSection.html:67]: The error message “There is no new selection.” is not working for all possible cases while not selecting  any teacher. It works on and off.

**Review by schoolmanagementproject2022 — APPROVED** [review]

**Pull request merged**